rebase.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Rebase On the Tree
+  hosts: localhost
+  become: yes
+  become_method: sudo
+
+  tasks:
+  - name: Create VM from Atomic QCOW2 image
+    command:
+      sh create-vm.sh mynode /path/to/f25-atomic.qcow2
+      # /path/to/f25-atomic.qcow2 = /var/lib/libvirt/images/Fedora-Atomic-25-20170131.0.x86_64.qcow2
+
+  - name: Get the IP of VM
+    command: virsh domifaddr mynode
+    register: VM_IP
+
+  - name: Create inventory file
+    file: path=inventory state=touch
+
+  - name: Write VM_IP in the inventory
+    lineinfile:
+      dest: inventory
+      line: '[atomichost]\n{{ VM_IP }} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic'
```

Comment:
@gbraad Need your help in here. I can get the IP address of the VM with the command `virsh domifaddr mynode`. But failing to write the `VM_IP` in the inventory. The inventory looks like this:

```
[atomichost]\n{u'changed': True, u'end': u'2017-02-06 16:55:18.563692', u'stdout': u' Name       MAC address          Protocol     Address\n-------------------------------------------------------------------------------\n vnet6      52:54:00:b1:fa:fe    ipv4         192.168.121.90/24', u'cmd': [u'virsh', u'domifaddr', u'mynode'], u'start': u'2017-02-06 16:55:18.544325', u'delta': u'0:00:00.019367', u'stderr': u'', u'rc': 0, 'stdout_lines': [u' Name       MAC address          Protocol     Address', u'-------------------------------------------------------------------------------', u' vnet6      52:54:00:b1:fa:fe    ipv4         192.168.121.90/24'], u'warnings': []} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic
```